Example/Tests/Classes/Utilities/LeanplumHelper.m

```diff
@@ -129,8 +124,8 @@ + (void)clean_up {
     [[LPAPIConfig sharedConfig] setToken:nil];
     [LeanplumRequest reset];
     [LeanplumHelper reset_user_defaults];
-    [[LeanplumRequest sendNowQueue] cancelAllOperations];
```

Comment:
So we already had a Queue correct? Why did we not use LeanplumRequest existing queue?

Reply:
I've moved `sendNowQueue` into its own class, the new `serialQueue` is also used by database to make sure that each event is actually written and sent in right order.